src/createTwitchVideoCompiler/youtubeUploader.js

```diff
@@ -0,0 +1,192 @@
+// @flow
+
+import fs from 'fs';
+import readline from 'readline';
+import { google } from 'googleapis';
+
+export default async function youtubeUploader(
+  clientFilePath: string,
+  properties: {
+    language: string,
+    description: string,
+    tags: string,
+    title: string,
+    privacy: string,
+    videoPath: string,
+  },
+): Promise<void> {
+  return new Promise((resolve, reject) => {
+    const { OAuth2 } = google.auth;
```

Comment:
Put this outside of the function. Two of the reasons are:
- This will be executed every time the function is called. It's unnecessary.
- Imagine someone updates the `googleapis` package and `google.auth` is no more. If this sits here, the program will crash when this function is called. If you put this outside of the function, the program will crash as soon as this file is included which would be easier to catch and fix.